app/webpacker/scripts/global/disable-browser-autofill.js

```diff
@@ -8,10 +23,27 @@ function disableBrowserAutofill () {
   }
 
   document.querySelectorAll('form').forEach(form => {
-    if (form.getAttribute('autocomplete') === 'off') {
-      const inputs = [...form.querySelectorAll('input[autocomplete]')].filter(input => inputTypes.includes(input.type))
+
+    let inputs = []
```

Comment:
Just a lil tidy up option! Lines 30 and 31 are repeated on lines 36 and 37, so I think you could change line 27 to:

```suggestion
  let inputs = [...form.querySelectorAll('input')].filter(input => inputTypes.includes(input.type))
```

and then inside your if statement becomes:

```
inputs.filter(input => input.dataset.jsDisableBrowserAutofill !== 'off')
```

then the same kind of simplification inside your else statement.